README.md

```diff
@@ -1,4 +1,12 @@
 ## ShortStrings
+| ⚠️ Notice ⚠️ |
+| --- |
+| **ShortStrings.jl is deprecated** |
+| [WeakRefString.jl's `InlineString`](https://github.com/JuliaData/WeakRefStrings.jl#inlinestring) is the same idea, but a more complete implementation. |
```

Comment:
```suggestion
| [InlineStrings.jl's `InlineString`](https://github.com/JuliaData/InlineStrings.jl) is the same idea, but a more complete implementation. |
```

Reply:
I split out the InlineString code into its own package: https://github.com/JuliaRegistries/General/pull/45335